workflows/methylation/methylation-cohort.wdl

```diff
@@ -247,10 +247,10 @@ task plot_umap {
     }
 
     runtime {
-        container: "ghcr.io/stjudecloud/python-plotting:2.0.1"
-        memory: "4 GB"
         cpu: 1
+        memory: "4 GB"
```

Comment:
I'd guess that this doesn't need the extra RAM, but we may not want to deal with it here.